src/NuGet.Core/NuGet.Build.Tasks/NuGet.targets

```diff
@@ -468,6 +468,7 @@ Copyright (c) .NET Foundation. All rights reserved.
      RestorePackagesPath="$(RestorePackagesPath)"
      RestoreFallbackFolders="$(RestoreFallbackFolders)"
      RestoreConfigFile="$(RestoreConfigFile)"
+     RestoreRootConfigDirectory="$(RestoreRootConfigDirectory)"
```

Comment:
do we wanna support this `RestoreRootConfigDirectory` via commandLine restore as well?

Reply:
What do you mean by commandline?
People can set the MSBuild property if they want to, because we can't really hide it. 
It's not called in the normal nuget.exe/dotnet.exe workflows.